### ISSUE TITLE
EDDI crashes on launch

### ISSUE THREAD

**Issue by PelvodelFuego** [EDCD/EDDI#218]
2.4.4 worked fine yesterday when I updated to it, but as of today it says the Frontier API connection is not operational and crashes ("EDDI has stopped working" Windows prompt)
![explorer_2017-11-08_17-49-46](https://user-images.githubusercontent.com/8196503/32542431-8ded4a34-c4ad-11e7-8b8a-d17f94cdb1e3.png)

Reverting to 2.4.3 has the same issue, which wasn't present before. 

2.4.2 works

Restarting the PC doesn't help. If someone can tell me how to back up my custom speech responder personality I'll try a full uninstall/reinstall. Otherwise let me know if I can help at all.

**Comment by richardbuckle** [comment]
Your custom speech responder personality is in `%APPDATA%\EDDI\personalities`, though you might want to separately backup the whole of `%APPDATA%\EDDI\`.

Before doing a full uninstall/reinstall, try removing `%APPDATA%\EDDI\credentials.json` -- that might be the problematic file.

**Comment by Tkael** [comment]
I've also had some scattered reports of %APPDATA%\EDDI\edsm.json causing an issue like this. If you remove that file, does that resolve the issue?

**Comment by PelvodelFuego** [comment]
Removing credentials.json alone didn't work so I reinstalled. Adding everything back to %APPDATA%\EDDI\ at once crashes it, but if I copy it all in one file at a time and test in between it works, so no single file seems to be a problem, no idea how that works.

In any case, I've disabled the EDSM plugin anyway because I run EDDiscovery too. It seems to be running fine for now, I'll update here if anything changes.
